lora/lorawan/region/au915.go

```diff
@@ -0,0 +1,22 @@
+package region
+
+import "tinygo.org/x/drivers/lora"
+
+type RegionSetAU915 struct {
+}
+
+func NewRegionSetAU915() RegionSetAU915 {
```

Comment:
This function could be named `AU915()` which would allow consumers to say `region.AU915()`